### ISSUE TITLE
Skip farm validation when rendering localy

### ISSUE THREAD

**Issue by mkolar** [ynput/ayon-nuke#11]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior:

When we try to publish local render in nuke, AYON still tried to validate connection to the render farm even though it's not really going to be used. If the farm was down, this would actually prevent using the local render option in the moment, when it's most usefull.

### Expected Behavior:

We should skip render farm validations when using local rendering.

### Version

1.0.0

### What platform you are running on?

Windows

### Steps To Reproduce:

1. Make sure deadline is configured in AYON to be used as render farm
2. Make sure no deadline webservice is running
3. Try to publish with local render option turned on in Nuke

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the bug report.

**Comment by mkolar** [comment]
This was migrated from https://app.clickup.com/t/6658547/AY-1474

**Comment by robin-ynput** [comment]
I cannot reproduce this issue, either I'm not understanding the request properly or this has already been fixed.
(I tried with ayon-deadline-0.24/ayon-nuke-0.2.3).

Repro steps:
1. Ensure deadline addons is part of the bundle and enabled but with no valid Deadline server
2. Open Nuke and create a "Local" Render
3. Publish it with with **Render target: Local machine rendering**
![image](https://github.com/user-attachments/assets/a9b8cfcb-bd1a-43e9-9110-ca403cac435c)
4. Publish goes through and Deadline Web service validation seems skipped
![image](https://github.com/user-attachments/assets/799b20fa-37a0-4690-b253-539980bdd669)
5. _Tried with  **Render target: Farm rendering** and it rightfully fails a Farm validation_
```
DEBUG: Starting new HTTP connection (1): 127.0.0.1:8082
Traceback (most recent call last):
  File "/home/robin/Desktop/dev_ayon/prod/AYON-1.0.3-linux/dependencies/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/home/robin/Desktop/dev_ayon/prod/AYON-1.0.3-linux/dependencies/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/home/robin/Desktop/dev_ayon/prod/AYON-1.0.3-linux/dependencies/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
```

So all looks good to me.

**Comment by jakubjezek001** [comment]
it was fixed via changes in 77b0d8c22575487f300ad95e4120345901e245d7
